packages/anon-aadhaar-pcd/src/utils.ts

```diff
@@ -261,3 +261,114 @@ export function decompressByteArray(byteArray: Uint8Array) {
   const decompressedArray = pako.inflate(byteArray)
   return decompressedArray
 }
+
+export const enum SELECTOR_ID {
+  emailOrPhone = 0,
+  referenceId,
+  name,
+  dob,
+  gender,
+  cafeof,
+  district,
+  landmark,
+  house,
+  location,
+  pinCode,
+  postOffice,
+  state,
+  stress,
+  subDistrict,
+  VTC,
+}
+
+export class SelectorBuilder {
+  selector: number[]
+
+  constructor() {
+    this.selector = new Array(16).fill(0)
+  }
+
+  selectEmailOrPhone() {
+    this.selector[0] = 1
+    return this
+  }
+
+  selectReferenceId() {
+    this.selector[1] = 1
```

Comment:
I think this don't need to be a selector? - we can simply extract the timestamp from this always?

Reply:
Yes, We know the position of email_or_phone_verifier and referenceID(last 4 digit and timestamp) so we can made it easy to query timestamp and last 4 digit